style.css

```diff
@@ -1,564 +1,566 @@
+  content: '';
+  height: 0.1rem;
+  width: 3.2rem;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: -1rem;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.feature-boxes {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.feature-box {
+  background-color: rgba(206, 212, 218, 0.1);
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  width: 100%;
+  padding: 1rem;
+}
+
+.feature-logo {
+  width: 4.8rem;
+}
+
+.feature-title {
+  color: var(--main-color);
+  font-size: 1.2rem;
+  font-weight: 400;
+  width: 20%;
+}
+
+.feature-desc {
+  color: #fff;
+  font-size: 1.2rem;
+  justify-self: end;
+  width: 60%;
+}
+
+.feature-btn1 {
+  padding: 2rem;
+  color: #fff;
+  background-color: var(--main-color);
+  border: none;
+  margin: 1.8rem 0;
+  cursor: pointer;
+}
+
+.feature-btn1:hover {
+  background-color: #a5392e;
+}
+
+.feature-btn2 {
+  display: none;
+}
+
+/* ***************** */
+
+/* **** SPEAKER ***** */
+
+/* ***************** */
+.speaker {
+  padding: 1.8rem 1.4rem;
+}
+
+.speaker-title {
+  color: #000;
+  font-size: 2.4rem;
+  font-weight: 400;
+  position: relative;
+  margin-bottom: 3.2rem;
+  text-align: center;
+}
+
+.speaker-title::after {
+  content: '';
+  height: 0.1rem;
+  width: 3.2rem;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: -1rem;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.speaker-box {
+  display: flex;
+  gap: 1.8rem;
+  margin-bottom: 1.8rem;
+}
+
+.speaker-img {
+  width: 11rem;
+}
+
+.speaker-details {
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+}
+
+.speaker-name {
+  font-size: 1.8rem;
+}
+
+.speaker-position {
+  font-size: 1.2rem;
+  color: var(--main-color);
+  position: relative;
+  margin-bottom: 1rem;
+  font-style: italic;
+}
+
+.speaker-position::after {
+  content: '';
+  height: 0.1rem;
+  width: 2rem;
+  background-color: #868e96;
+  position: absolute;
+  bottom: -1rem;
+  left: 0;
+}
+
+.speaker-desc {
+  line-height: 1.5;
+}
+
+/* ***************** */
+
+/* **** PARTNERS ***** */
+
+/* ***************** */
+
+.partner {
+  background-color: #414246;
+  padding: 3.2rem;
+  display: none;
+}
+
+.partner-title {
+  color: #8f8f91;
+  font-size: 2.4rem;
+  font-weight: 400;
+  position: relative;
+  margin-bottom: 3.2rem;
+  text-align: center;
+}
+
+.partner-title::after {
+  content: '';
+  height: 0.1rem;
+  width: 3.2rem;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: -1rem;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.partners {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.partners img {
+  height: 5rem;
+  padding: 0.4rem;
+  filter: brightness(2);
+  opacity: 30%;
+}
+
+/* ***************** */
+
+/* **** FOOTER ***** */
+
+/* ***************** */
+.footer {
+  padding: 2rem 3rem;
+  display: flex;
+  justify-content: space-between;
+  gap: 2.4rem;
+  align-items: center;
+}
```

Comment:
- [ ] **[OPTIONAL]** Fantastic work on implementing the suggestion provided **by the previous reviewer in your project**. I would like to suggest another **optional improvement that could enhance the visual appeal and professionalism even further**. Have you considered changing the image? Currently, it appears small and confined. By making this change, you would create a more visually appealing and polished look for your project. 🎯

Reply:
@taldr27 Done making changes.